### PR TITLE
Completed css-colors exercise. - Henry Ding

### PR DESCRIPTION
**Pull request by henryrding**
image

### CODE REVIEW

File: css-colors/styles.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: aliceblue;
```
Comment (shawnkost) [inline css-colors/styles.css:8]: The `background-color` property should be applied to the `section `rather than the `body`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Henry! One small change requested here

File: css-colors/styles.css (hunk 1 above)
Comment (henryrding) [reply css-colors/styles.css]: image

**Review by shawnkost — APPROVED** [review]
:+1:

**Pull request merged**